server/index.js

```diff
@@ -1,21 +1,27 @@
 import {setupApp} from './setup-app';
 import {createPageConfig} from './model/page-config';
 import {default as filtersMap} from './filters';
+import Prismic from 'prismic-javascript';
 import {
   getPrismicApi,
   getEvent
 } from './services/prismic';
 import {
   parsePromoListItem,
-  parseExhibitionsDoc
+  parseExhibitionsDoc,
+  prismicImage,
+  asText
 } from './services/prismic-parsers';
 
 export {setupApp};
 export const filters = filtersMap.toJS();
 export const model = {createPageConfig};
 export const prismicParsers = {
   parsePromoListItem,
-  parseExhibitionsDoc
+  parseExhibitionsDoc,
+  prismicImage,
+  asText
 };
+export {Prismic};
```

Comment:
See here:
https://github.com/wellcometrust/wellcomecollection.org/pull/1879#discussion_r154101728

Reply:
You might have to talk me through what you're thinking here – why, e.g., are we exporting two prismic-related services as separate things? https://github.com/wellcometrust/wellcomecollection.org/blob/master/server/index.js#L20-L21

Reply:
The reason for this I suppose is that we;re retrospectively doing it, so should be retrospectively trying to clean up to.

That's my question here - we could export the root Prismic object and do all the predicate stuff in the specific services as @gestchild and you've done, but my thinking was to create the Prismic service, that abstracts that out and exports it to the exhibition service, which then only has to deal with the `Exhibition` type.

the point of having the Prismic service, is exactly that, making it a service.

it might also land up with optimisations being made once that could propagate to it's dependants, but that's a maybe-one-day...

Reply:
See here for instance, where an abstraction in that service could propagate elsewhere saving time...
https://github.com/wellcometrust/wellcomecollection.org/pull/1879#discussion_r154880114